plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -146,51 +162,27 @@ class ServiceDestroyModal extends React.Component {
     );
   }
 
-  getDestroyFrameworkModal() {
-    const { open, service, intl } = this.props;
-    const packageName = service.getPackageName();
+  getGroupHeader() {
+    if (!this.isGroupWithServices()) {
+      return;
+    }
 
     return (
-      <Modal
-        header={this.getModalHeading()}
-        footer={this.getCloseButton()}
-        modalClass="modal"
-        onClose={this.handleModalClose}
-        open={open}
-        showHeader={true}
-        showFooter={true}
-        subHeader={this.getSubHeader()}
-      >
+      <div className="modal-service-delete-center">
         <p>
-          {intl.formatMessage({
-            id: "SERVICE_ACTIONS.DELETE_SERVICE_FRAMEWORK"
-          })}
-          <a
-            href="https://docs.mesosphere.com/service-docs/"
-            target="_blank"
-            title={intl.formatMessage({
-              id: "COMMON.DOCUMENTATION_TITLE"
-            })}
-          >
-            {intl.formatMessage({
-              id: "COMMON.DOCUMENTATION"
-            })}
-          </a>
-          {intl.formatMessage({
-            id: "SERVICE_ACTIONS.DELETE_SERVICE_FRAMEWORK_2"
-          })}
+          This group needs to be empty to delete it. Please delete any services in the group first.
```

Comment:
This is a bit confusing/misleading. It's not true that it needs to be empty to delete it. It's only true that it needs no SDK service to delete it. Maybe change to:

"Cannot delete group containing SDK services. Please delete these services before deleting the group."

Reply:
Actually, we want to keep it generic like this so we don't have to call out specific types of services that you can/cannot delete etc.

Reply:
Hm, okay fair enough. 👍

Reply:
Sounds good, @ahoskins if you could remove the block and/ or approve the PR if we don't have any more issues, that would be greeeeat!